src/sentry/static/sentry/app/components/events/contexts/trace/trace.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import {Event, Organization} from 'app/types';
+import {t} from 'app/locale';
+import withOrganization from 'app/utils/withOrganization';
+import ErrorBoundary from 'app/components/errorBoundary';
+import KeyValueList from 'app/components/events/interfaces/keyValueList/keyValueListV2';
+
+import {TraceKnownData, TraceKnownDataType} from './types';
+import getTraceKnownData from './getTraceKnownData';
+
+const traceKnownDataValues = [
+  TraceKnownDataType.STATUS,
+  TraceKnownDataType.TRACE_ID,
+  TraceKnownDataType.SPAN_ID,
+  TraceKnownDataType.PARENT_SPAN_ID,
+  TraceKnownDataType.TRANSACTION_NAME,
+  TraceKnownDataType.OP_NAME,
+];
+
+type Props = {
+  organization: Organization;
+  event: Event;
+  data: TraceKnownData;
+};
+
+const InnerTrace = withOrganization(function({organization, event, data}: Props) {
+  return (
+    <ErrorBoundary mini>
+      <KeyValueList
+        data={getTraceKnownData(data, traceKnownDataValues, event, organization)}
+        isSorted={false}
+        raw={false}
+      />
+    </ErrorBoundary>
+  );
+});
+
+const Trace = (props: Props) => {
+  return <InnerTrace {...props} />;
+};
+
+Trace.getTitle = function() {
```

Comment:
I've updated this in 'contexts.jsx', creating the method ' getTitle', so I believe you could move it to this method

Reply:
Thanks for the heads up. Addressed in https://github.com/getsentry/sentry/pull/19078/commits/51f613d9aa56b39ce708e4f78afb6fc69b6c92ee